src/components/EventItem.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const EventItem = () => (
+  <div>
+    event item
+    <div>photo</div>
+    <h1>title</h1>
```

Comment:
Be careful with `h` tags. You should have only one `h1` per route/view. And don't remember that you have more `h` tags then only `h1` :)